tests/wallet/test_wallet.py

```diff
@@ -85,6 +85,7 @@ def test_wallet_create_transaction(self):
         tx1.storage = self.storage
         tx1.update_hash()
         tx1.get_metadata().validation = ValidationState.FULL
+        tx1.init_static_metadata_from_storage(self._settings, self.storage)
```

Comment:
Same reason as above, since this is only used in tests, it's possible that some tests use this method and don't need static metadata to be calculated. Also it's together with `update_hash` and `get_metadata`.

Reply:
Ok!